### PR TITLE
Update contributer list

### PR DESCRIPTION
**Pull request by Hopekz**

### CODE REVIEW

**Comment by boodaliboo** [comment]
you added a spelling mistake

**Comment by Hopekz** [comment]
> you added a spelling mistake

Okay; want to point it out or are you going to keep me guessing?

**Comment by ThatLing** [comment]
This list is still so stupid

**Comment by JamieD1** [comment]
Where the hell am I

**Comment by AsV9** [comment]
@ThatLing Think there's a way to pull members of an org team using the API, would be kinda funny to set that up tbh

**Comment by ThatLing** [comment]
@AsV9 Should just make a page on the wiki with the current headcoders and maintainers (isn't there already a page like this?) and link to it. Making a manual list for "mappers and spriters" is stupid.

EDIT:
https://wiki.yogstation.net/wiki/Administrators#Contributors

File: html/changelog.html [hunk 1]
```diff
@@ -37,9 +37,9 @@
 		<td valign='top'>
 			<font size='2'><b>Current Headcoders:</b>Alexkar598, Monster860<br></font>
 			<font size='2'><b>Currently Active GitHub contributor list:</b> <a href='https://github.com/yogstation13/Yogstation/graphs/contributors'>-Click Here-</a><br></font>
-			<font size='2'><b>Maintainers:</b> Alexkar598, AshCorr, Fluffe9911, Identification, Ktlwjec, McDonald072, Monster860, Partheo, ThatLing, Theos<br></font>
```
Comment (ThatLing) [inline html/changelog.html:40]: McDonald072 is still a maintainer

**Review by ThatLing — DISMISSED** [review]

**Comment by Hopekz** [comment]
> Where the hell am I

Where do you belong on the list?

File: html/changelog.html (hunk 1 above)
Comment (Hopekz) [reply html/changelog.html:40]: The more you know

**Comment by JamieD1** [comment]
> > Where the hell am I
> 
> Where do you belong on the list?

I want my own line.

**Comment by Hopekz** [comment]
> > > Where the hell am I
> > 
> > 
> > Where do you belong on the list?
> 
> I want my own line.

Out of scope for this PR

**Review by nesisknesis — COMMENTED** [review]
remove r3d from spriters, add Tau Ceti Station, CEV-Eris, and BeeStation to the thanks to

**Comment by JamieD1** [comment]
Reeee RIP Coders also im a mapper too!

**Review by SomeguyManperson — APPROVED** [review]

**Pull request merged**